en/publish/publish-vivo-instant-games.md

```diff
@@ -1,140 +1,146 @@
 
-- **小包模式和小包模式服务器路径**
+- **Small package mode and packet mode server path**
 
-  该项为选填项。小游戏的包内体积包含代码和资源不能超过 4M，资源可以通过网络请求加载。**小包模式** 就是帮助用户将脚本文件保留在小游戏包内，其他资源则上传到远程服务器，根据需要从远程服务器下载。而远程资源的下载、缓存和版本管理，Creator 已经帮用户做好了。用户需要做的是以下两个步骤：
+  This item is optional. The in-package volume of the mini-game contains code and resources that cannot exceed 4M, and resources can be loaded via network requests. **Small package mode** is to help users keep the script files in the small game package, other resources are uploaded to the remote server, and downloaded from the remote server as needed. And the download, cache and version management of remote resources, __Cocos Creator__ has already helped the user. What the user needs to do is the following two steps:
 
-  1、构建时，勾选 **小包模式**，填写 **小包模式服务器路径**。然后点击 **构建**。
+  1. When building, check the **packet mode** and fill in the **packet mode server path**. Then click on **Build**.
 
-  2、构建完成后，点击 **发布路径** 后面的 **打开** 按钮，将发布路径下的 **jsb-link/res** 目录上传到小包模式服务器。例如：默认发布路径是 build，则需要上传 build/jsb-link/res 目录。
+  2. After the build is complete, click the **Open** button after the **Publish Path** to upload the **jsb-link/res** directory under the release path to the packet mode server. For example, if the default publishing path is build, you need to upload the **build/jsb-link/res** directory.
 
-  此时，构建出来的 qgame 目录下将不再包含 res 目录，res 目录里的资源将通过网络请求从填写的 **小包模式服务器地址** 上下载。
+  At this point, the built-up `qgame` directory will no longer contain the `res` directory, and the resources in the `res` directory will be downloaded from the filled **packet mode server address** via the network request.
 
-二、**构建发布** 面板的相关参数设置完成后，点击 **构建**。构建完成后点击 **发布路径** 后面的 **打开** 按钮来打开构建发布包，可以看到在默认发布路径 build 目录下生成了 **qgame** 目录，该目录就是导出的 vivo 小游戏工程目录。
+Second, **Build release** After the relevant parameters of the panel are set, click **Build**. After the build is complete, click the **Open** button behind the **Publish Path** to open the build release package. You can see that the **qgame** directory is generated under the default release path build directory, which is the exported __Vivo Mini Game__ project directory.
 
 ![](./publish-vivo-instant-games/package.png)
 
-三、生成 rpk 包（如果在 **构建发布** 面板有填写了 **本地 npm 安装路径**，就可以跳过该步骤）。
+Third, generate the rpk package (if you have filled out the **local npm installation path** in the **Build Release** panel, you can skip this step).
 
 ```bash
-# 命令行指定到 qgame 目录下
+# Command line specified to the qgame directory
 cd E:\workspace\YourProject\build\qgame
 npm install
 npm run build
 ```
 
-然后在 /build/qgame/dist 目录下就会生成 rpk 包。
+Then the rpk package will be generated in the **/build/qgame/dist** directory.
 
 ![](./publish-vivo-instant-games/rpk.png)
 
-四、将打包出来的 rpk 运行到手机上。有以下两种方式可将 rpk 运行到手机上：
+4. Run the packaged rpk to your phone. There are two ways to run rpk on your phone:
```

Comment:
The first step is "1", the second step is "Second", the third step is "Third", the four step is "4". And maybe we need to unify the way we write it?